docs/deployment-guide.md

```diff
@@ -213,20 +213,123 @@ If you have a valid license place your `.lic` file in the `configuration_files/l
 
 > NOTE: You can also [upload a license](https://docs.alfresco.com/content-services/latest/admin/license/) via the Admin Console once the system is running.
 
-### Alfresco/Solr authentication
+### Secrets management
 
-As of ACS 7.2 and/or Search services 2.0.3, the repository <--> solr communication requires to be authenticated. The playbook will set up that authentication scheme using the new `secret` method.
-This methods needs to be passed a shared secret. In order to do so use the variable below:
+This playbook expects that security-relevant variables are configured within the
+`vars/secrets.yml` file.
 
-```yaml
-reposearch_shared_secret: dummy
+It is strongly recommended to enable [Ansible
```

Comment:
Not clear if Ansible vault is a recommendation or a prerequisite.

In order to avoid keeping secrets in plain text on the control node It is strongly recommended to enable [Ansible
 Vault](https://docs.ansible.com/ansible/latest/user_guide/vault.html) encryption. Though [third-party plugins](#third-party-lookup-plugins) can be used, the rest of the guide provide example for Ansible Vault.
 or use .

Reply:
credentials in secrets.yml are mandatory, while vault is not